documentation/docs/quickstart/3-configure-bot.md

```diff
@@ -26,8 +23,25 @@ bounty  Participate in hummingbot's liquidity bounty programs
 
 etc...
 ```
+## Step 2: Enable Paper Trading Mode (Optional)
+
+You can run Hummingbot and simulate trading strategies without executing and placing actual trades. Run command `paper_trade` at the beginning to enable this feature.
+
+```
+Enable paper trading mode (y/n) ? >>> y
+
+New config saved:
+paper_trade_enabled: y
+
+Your configuration is incomplete. Would you like to proceed and finish all necessary configurations? (y/n) >>> y
```

Comment:
this should be `n`, since the user will be configuring the bot in step 4 below, not here.